### PR TITLE
remove gboard learning

### PR DESCRIPTION
**Pull request by jpcloureiro**

### CODE REVIEW

**Comment by jpcloureiro** [comment]
No dependencies.

Can be merged once approved

**Review by sethkfman — APPROVED** [review]
LGTM

**Comment by cortisiko** [comment]
https://github.com/MetaMask/mobile-planning/issues/1436#issuecomment-2019116176 ✅

**Pull request merged**
